app/webpack/stylesheets/gto/_spinner.scss

```diff
@@ -1,22 +1,23 @@
 @import "colors";
+$base-size: 6.4;
```

Comment:
I didn't want to move all the fixed em things we had but this wouldn't scale so ... hopefully this doesn't bust a bunch of other spinners :D